lib/slack-rtmapi.rb

```diff
@@ -17,4 +23,39 @@ def self.get_url(options)
     URI(body['url'])
   end
 
+  def initialize(options)
+    @token = options.fetch(:token)
+    @api_url = options[:slack_api_url] || 'https://slack.com/api'
+  end
+
+  def start_rtm
+    @data = SlackWebAPI.call(method: 'rtm.start', token: @token)
+    @websocket_url = URI(@data['url'])
+    client
+  end
+
+  def client
+    @client ||= SlackRTM::Client.new websocket_url: @websocket_url
+  end
+
+  def channel_id(channel_name)
+    channels = @data['channels']
+    channel_data = channels.find { |channel| channel['name'] == channel_name }
+    channel_data['id'] unless channel_data.nil?
+  end
```

Comment:
Yeah, I would also like to implement this kind of feature, but not like that (because one of my use case use a ActiveRecord storage for this kind of data, and another use Redis).
I think we would need to use a proxy which can back a simple hash, an activerecord or another thing.

Anyway, i don't think that this method should be in the `SlackRTM` module, as the purpose of this module is **only** to wire the base communication. Storage is the responsibility of another module (or group of modules).